Editor/Gui/Windows/ScreenManager.cs

```diff
@@ -0,0 +1,441 @@
+using ImGuiNET;
+using Microsoft.Win32;
+using System.Diagnostics;
+using System.Drawing;
+using System.Windows.Forms;
+using T3.Core.Utils;
+using T3.Editor.App;
+using T3.Editor.Gui.Input;
+using T3.Editor.Gui.Styling;
+using T3.Editor.Gui.UiHelpers;
+using T3.Editor.Gui.Windows.Layouts;
+using Icon = T3.Editor.Gui.Styling.Icon;
+using Vector2 = System.Numerics.Vector2;
+
+namespace T3.Editor.Gui.Windows;
+
+internal sealed class ScreenManager : Window
+{
+    internal ScreenManager()
+    {
+        Config.Title = "Screen Manager";
+        SystemEvents.DisplaySettingsChanged += (_, __) => _layoutDirty = true;
```

Comment:
Careful: If you subscribe to an event like, this you normally would need to implement `IDispose` to clean this up. Here it's not really a problem, because there is only one Window. But event handlers can be tricky and ask to be used with care.